src/main/resources/sql/populate_data.sql

```diff
@@ -25,28 +25,19 @@ VALUES
 
 
 INSERT INTO
- cdss_decoupling.users (username, name, password, enabled, role) 
+ cdss_decoupling.users (username, name, password, enabled, role, supplierId)
 VALUES
- ('admin', 'Admin User', '$2a$10$hbxecwitQQ.dDT4JOFzQAulNySFwEpaFLw38jda6Td.Y/cOiRzDFu', true, 'ROLE_ADMIN');
+ ('admin', 'Admin User', '$2a$10$hbxecwitQQ.dDT4JOFzQAulNySFwEpaFLw38jda6Td.Y/cOiRzDFu', true, 'ROLE_ADMIN', 'admin');
```

Comment:
just a heads up we've removed this file as it's data.sql in the level above that runs. We've added the line to add 'admin' supplierId in our branch.